### PR TITLE
Announcements with media

### PR DESCRIPTION
**Pull request by CunningBard**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

This PR adds a Media Announcement Command, Which lets you announcement with specific media, this fulfills feature #27 

Fixes # (issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: bot/cogs/announcements.py [hunk 1]
```diff
@@ -1,15 +1,52 @@
...
+    )
+    @describe(
+        photo_link="The Link to the Photo",
+        to_announce="Message to announce"
+    )
+    @checks.has_any_role(
+        "Admin",
+        "Moderator",
+        "Helper",
+        "baby, ako na lang kasi"
```
Comment (Jedddy) [inline bot/cogs/announcements.py:25]: Hoy HAHAHA

**Review by Jedddy — DISMISSED** [review]
LGTM 👍

File: bot/cogs/announcements.py (hunk 1 above)
Comment (CunningBard) [reply bot/cogs/announcements.py:25]: leave it as a easter egg, It wont break the bot

Comment (Jedddy) [reply bot/cogs/announcements.py:27]: oke HAHHAA

**Review by Jedddy — APPROVED** [review]
yes lgtm

**Review by tora-o — APPROVED** [review]

**Pull request merged**